File: src/main/java/de/tectoast/emolga/commands/various/DasorCommand.java
package de.tectoast.emolga.commands.various;

import de.tectoast.emolga.commands.Command;
import de.tectoast.emolga.commands.CommandCategory;
import de.tectoast.emolga.commands.GuildCommandEvent;
import de.tectoast.emolga.utils.sql.DBManagers;

public class DasorCommand extends Command {


    public DasorCommand() {
        super("dasor", "ist cool", CommandCategory.Various);
        setArgumentTemplate(ArgumentManagerTemplate.noArgs());
    }

    @Override
    public void process(GuildCommandEvent e) throws Exception {
        e.reply(DBManagers.DASOR_USAGE.buildMessage());
    }
}

File: src/main/kotlin/de/tectoast/emolga/commands/various/DasorCommand.kt
package de.tectoast.emolga.commands.various

import de.tectoast.emolga.commands.Command
import de.tectoast.emolga.commands.CommandCategory
import de.tectoast.emolga.commands.GuildCommandEvent
import de.tectoast.emolga.database.exposed.DasorUsageDB

class DasorCommand : Command("dasor", "ist cool", CommandCategory.Various) {
    init {
        argumentTemplate = ArgumentManagerTemplate.noArgs()
    }

    override suspend fun process(e: GuildCommandEvent) {
        e.reply(DasorUsageDB.buildMessage())
    }
}
